src/pages/index.js

```diff
@@ -20,11 +20,19 @@ const IndexPage = () => (
       strength={400}
     >
       <div className="image-shadow text-center py-32 text-white" >
-        <img src={logo} />
-        <h1 className="m-0 font-bold"> Just Tech </h1>
-        <p className="m-0"> Matchar utrikesfödda topptalanger </p>
-        <p className="m-0"> inom IT med arbetsmarknaden </p>
-        <p className="m-0"> Start Q1 2019 </p>
+        <img src={logo} alt="Just logo"/>
+
+        <h1 className="m-0"> Just<b>Tech</b> </h1>
+
+        <p className="m-4">
+          Matchar utrikesfödda topptalanger<br/>
+          inom IT med arbetsmarknaden
+        </p>
+        <p className="m-4">
+          <b>Start Q1 2019</b>
+        </p>
+
+        <div class="m-3" style={{background: "white", height: "8px", margin: "auto", width: "24px"}}></div>
```

Comment:
We can use Tailwind built-in classes, 

```
{background: "white"} becomes "bg-white"
{margin: "auto"} -> m-aouto
```
and for CSS units its better to use ``` rem ``` in place of ```px```
as font-size defaults to ```16px```, ```8px``` become ```0.5rem``` then we can use Tailwind`s ```h-2``` and ```w-6``` :dancer: 
at the end we will have:
`<div class="m-auto bg-white h-2 w-6" ></div>`

Reply:
Cool, I'll try it out. Thanks for the feedback!

Reply:
Your welcome :)